.github/workflows/test.yml

```diff
@@ -10,7 +10,7 @@ on:
 jobs:
   checks:
     name: run
-    runs-on: ubuntu-latest
+    runs-on: self-hosted
```

Comment:
We should actually use 
`runs on [self-hosted, dpdk]`
Please check how this is done in metalnet and please do in exact same way.